lib/zip/entry_set.rb

```diff
@@ -21,13 +21,13 @@ def <<(entry)
       @entry_set[to_key(entry)] = entry
     end
 
-    alias :push :<<
+    alias_method :push, :<<
 
     def size
       @entry_set.size
     end
 
-    alias :length :size
+    alias_method :length, :size
```

Comment:
There is no sense to use alias_method because we don't need chains. So better use alias instead of alias_method